model/common/src/icon4py/model/common/test_utils/serialbox_utils.py

```diff
@@ -402,7 +402,7 @@ def _read_field_for_dim(field_name, read_func, dim: gtx.Dimension):
                     f"only {dims.CellDim, dims.EdgeDim, dims.VertexDim} are handled"
                 )
 
-    def owner_mask(self, dim: gtx.Dimension):
+    def _owner_mask(self, dim: gtx.Dimension):
```

Comment:
why would you make it private. I would not discourage direct access to these fields. They might be used in tests without going through the `DecompositionInfo`.